### ISSUE TITLE
save permission without any intent filter

### ISSUE THREAD

**Issue by kamrankhorasani** [felHR85/UsbSerial#259]
is there any solution to save usb host permission for next use of usb device without any intent filter?

**Comment by dipakbari4** [comment]
Yes, I think you should have to use SharedPreference and pass the data on next use from shared preference.

**Comment by kamrankhorasani** [comment]
> Yes, I think you should have to use SharedPreference and pass the data on next use from shared preference.

i want set permission first time for some usb device and connect to them and for second time when i plug in my usb device i dont want request any permission for this usb and i dont want using any intent filter . you said save data to shared preference . so what data should i save and how to use them to prevent any permission request for usb device?

**Comment by dariocavada** [comment]
> > Yes, I think you should have to use SharedPreference and pass the data on next use from shared preference.
> 
> i want set permission first time for some usb device and connect to them and for second time when i plug in my usb device i dont want request any permission for this usb and i dont want using any intent filter . you said save data to shared preference . so what data should i save and how to use them to prevent any permission request for usb device?

Any advice on that ?
